OptimizelySDKUniversal/generated-frameworks/Release-iOS-universal-SDK/OptimizelySDKiOS.framework/Headers/OPTLYProjectConfig.h

```diff
@@ -45,7 +45,7 @@ NS_ASSUME_NONNULL_END
 /// Datafile Revision number
 @property (nonatomic, strong, nonnull) NSString *revision;
 /// Flag for IP anonymization
-@property (nonatomic, assign) BOOL anonymizeIP;
+@property (nonatomic) NSNumber<Optional> *anonymizeIP;
```

Comment:
why does this need to be changed in two places?

Reply:
You are looking at build content in the autogenerated "generated-frameworks"
subfolder.  The original human maintained source code is in
OptimizelySDKCore/OptimizelySDKCore/OPTLYProjectConfig.h .  The
duplication occurs because more than one build product copies the
original human file.